pom.xml

```diff
@@ -215,6 +215,9 @@
 				<configuration>
 					<!-- The package of your generated sources -->
 					<packageName>tds.exam.results.trt</packageName>
+                    <sources>
+                        <source>src/main/resources/xsd</source>
```

Comment:
I'm a little confused why this is required... Shouldn't adding the xsd to src/main/resources automagically add it to the classpath?